### PR TITLE
DR-1877 Helm setup for RBS

### PR DESCRIPTION
**Pull request by tlangs**
Setting the values for RBS in dev, integration, and perf.

### CODE REVIEW

**Review by samanehsan — APPROVED** [review]
Lgtm! 🌤️

File: integration/integration-4/datarepo-api.yaml [hunk 1]
```diff
@@ -13,6 +13,9 @@ env:
   IT_JADE_API_URL: https://jade-4.datarepo-integration.broadinstitute.org
   AZURE_CREDENTIALS_APPLICATIONID: 22cb243c-f1a5-43d8-8f12-6566bcce6542
   AZURE_CREDENTIALS_HOMETENANTID: fad90753-2022-4456-9b0a-c7e5b934e408
+  RBS_ENABLED: true
+  RBS_POOL_ID: datarepo_v1
+  RBS_INSTANCE_URL: https://buffer.tools.integ.envs.broadinstitute.org
```
Comment (nmalfroy) [inline integration/integration-4/datarepo-api.yaml:18]: int 4 and 5 shouldn't use tools since objects in there are persistent

**Review by nmalfroy — CHANGES_REQUESTED** [review]
looks good but one comment about int4 and int5 needing to be persistent

File: integration/integration-4/datarepo-api.yaml (hunk 1 above)
Comment (tlangs) [reply integration/integration-4/datarepo-api.yaml:18]: oh... Will update!

**Review by nmalfroy — APPROVED** [review]
looks good!

**Pull request merged**